docs/reference/avalanchego/info-api.md

```diff
@@ -22,6 +22,97 @@ This API uses the `json 2.0` RPC format. For more information on making JSON RPC
 
 ## Methods
 
+### `info.acps`
+
+Returns peer votes for Avalanche Change Proposals (ACPs)
+
+**Signature:**
+
+```sh
+info.acps() -> {
+    acps: map[uint32]{
+        supportWeight: uint64
+        supporters:    set[string]
```

Comment:
I don't know what format/language this is supposed to be, so let me know if there's a correct way to represent a set data type